tables/cost_and_usage_report/cost_and_usage_report_extractor.go

```diff
@@ -132,6 +132,23 @@ func extractFromCSV(reader io.Reader) ([]any, error) {
 		record := &CostUsageReport{}
 		record.MapValues(recordMap)
 
+		// We should have null value if the map value is empty
+		if len(*record.Product) == 0 {
```

Comment:
What does the data look like if `Product` is empty in the CUR? Is it `{}`?

Reply:
Yes, in the CSV report, it appears as `{}`. Generally, if the column is of JSON type, an empty value is represented as `{}`.